### ISSUE TITLE
Reduce duplication in Java SDK documentation

### ISSUE THREAD

**Issue by mraible** [stormpath/stormpath-sdk-java#889]
There are many duplicate images in the following directories. Figure out a way to make the documentation DRY so only one image exists if it's the same.

* docs/source/_static/
* extensions/spring/boot/docs/source/_static/
* extensions/spring/stormpath-spring-security/docs/source/_static/

**Comment by mraible** [comment]
Depends on #504. Outcome of that issue's resolution could reduce duplication.

**Comment by jarias** [comment]
https://github.com/stormpath/stormpath-sdk-java/issues/949 might fix this

**Comment by mraible** [comment]
@jarias You're right. This a duplicate of #949. I'll close it.
